content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -16,21 +16,49 @@ Credentials tab
 
 Odoo needs your **API Credentials** to connect with your Stripe account, which comprise:
 
-- Publishable Key: The key solely used to identify the account with Stripe.
-- Secret Key: The key to sign the merchant account with Stripe.
-- Webhook Signing Secret: If a webhook is enabled on your Stripe account
-  (:menuselection:`Developers --> webhooks`), this signing secret must be set to authenticate the
-  messages sent from Stripe to Odoo.
-
-To retrieve the publishable and secret keys, log into your Stripe dashboard and go to
-:menuselection:`Developers --> API Keys --> Standard Keys`
+- :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
+- :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
+- :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
+  account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
 
 .. important::
    If you are trying Stripe as a test, in the **test mode**, change the **State** to *Test
    Mode*. We recommend doing this on a test Odoo database, rather than on your main database.
 
+.. _stripe/api_keys:
+
+Publishable and Secret keys
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve the publishable and secret keys, follow this `link to your api keys
+<https://dashboard.stripe.com/account/apikeys>`_, or log into your Stripe dashboard and go to
+:menuselection:`Developers --> API Keys --> Standard Keys`.
+
+.. _stripe/webhook:
+
+Webhook Signing Secret
+~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve the webhook signing secret, you first need to create a webhook. To do so, follow
+this `link to your webhooks <https://dashboard.stripe.com/webhooks>`_, or log into your Stripe
+dashboard and go to :menuselection:`Developers --> Webhooks`.
+
+Then, click on **Add endpoint** in your **Hosted endpoints**. A form opens, where you'll need to add
+two data:
+
+- In the **Endpoint URL**, enter ``https://yourcompany.odoo.com/payment/stripe/webhook`` after
+  replacing *yourcompany.odoo.com* by your address.
+- At the end of the form, you can **Select events** to listen to. Click on it and, in the
+  **Checkout** section, select **checkout.session.completed**.
+
+.. note::
+  It is possible to select other events but they are currently not processed by Odoo.
+
+When you click on **Add endpoint**, your Webhook will be configured. You can then click on
+**reveal** to see your signing secret.
```

Comment:
```suggestion
When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
display your signing secret.
```
- present tense
- see --> display
- break the ligne only after the 100th character. see  [RST guidelines](https://www.odoo.com/documentation/14.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character)